rocky/reports/report_types/aggregate_organisation_report/report.py

```diff
@@ -80,9 +80,9 @@ def post_process_data(self, data: dict[str, Any], valid_time) -> dict[str, Any]:
 
                         for service in system["services"]:
                             if service not in services:
-                                services[service] = {str(ip): systems["services"][ip]}
+                                services[service] = {ip: systems["services"][ip]}
```

Comment:
If I remember correctly the `str` was added here because the JSON export breaks if you have a dictionary has keys that aren't string or other simple types.

Reply:
I've done it this way to be able to show the human readable IP in the HTML (there's no other way that won't break other parts of the reports). Tested the JSON output, works perfectly. The JSON output is still the same.